### DISCUSSION TITLE
User selectable number of rows displayed in tables

### DISCUSSION THREAD

**Discussion by lonnylundsten** [datapane/datapane#366] [Ideas]
### Goals

1. I would like to display more than 10 rows of data in the DataTable block
2. I realize all the data is available by scrolling but in some instances I would like to display a table with no plots or images and when limited to 10 there is too much wasted space on the web page

### Non-Goals

1.
2.
3.

### Background

I don't see other options for doing this aside from using Table.

### Proposal

Programmer should be able to select the rows displayed in the DataTable block.

**Comment by mands** [comment]
Agreed - picking up
